sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/encoders/EncoderResolutionSuite.scala

```diff
@@ -54,38 +54,38 @@ class EncoderResolutionSuite extends PlanTest {
     val encoder = ExpressionEncoder[StringLongClass]
 
     // int type can be up cast to long type
-    val attrs1 = Seq('a.string, 'b.int)
+    val attrs1 = Seq(Symbol("a").string, Symbol("b").int)
```

Comment:
If we replace this with another semantically-equal one, I prefer $"a" instead of `Symbol` as for the SQL module. IIRC @cloud-fan left the same comment in another PR.